koji/copy-tags-commands.sh

```diff
@@ -10,11 +10,11 @@ fi
 OLD=$1
 VERSION=$2
 
-NONSCL_SYSTEMS="fedora24"
+NONSCL_SYSTEMS=""
 SCL_SYSTEMS="rhel7"
 
 clone() {
-  echo kkoji clone-tag $PRODUCT-$OLD-$SYSTEM $PRODUCT-$VERSION-$SYSTEM
+  echo kkoji clone-tag --config $PRODUCT-$OLD-$SYSTEM $PRODUCT-$VERSION-$SYSTEM
```

Comment:
Newer versions of koji require --config to be passed so that the config is copied from the old to the new tag